src/internal/operations/backup.go

```diff
@@ -36,9 +36,10 @@ type BackupOperation struct {
 
 	ResourceOwner common.IDNamer
 
-	Results   BackupResults      `json:"results"`
-	Selectors selectors.Selector `json:"selectors"`
-	Version   string             `json:"version"`
+	Results       BackupResults      `json:"results"`
+	Selectors     selectors.Selector `json:"selectors"`
+	Version       string             `json:"version"`
+	BackupVersion int                `json:"backupVersion"`
```

Comment:
beyond that, I'm a little on the fence about having the backup operation keep the version that it's using. This assumes we can always determine the version up-front when the operation is created, before even examining what it is we're supposed to backup (i.e. Exchange, SharePoint) and how (if we have persisted feature flags on backups in the future, what feature flags were enabled on the base, etc).

Overall, I think it works well to get the test running, but I'm not sure if it will allow us to continue to expand Corso as a system without revisiting it. To pitch an alternative for getting testing to work, one thing that's possible is to add a wrapper mock around the call to get previous backups and modify things on the fly there. It does require more setup and such, but would allow us to delay the point where we actually need to say what version we want to place in the backup model

The above is more a discussion point rather than something that has to be done in this PR, but I think moving forward it's something we're going to have to figure out how to address

Reply:
> Overall, I think it works well to get the test running, but I'm not sure if it will allow us to continue to expand Corso as a system without revisiting it.

Yep.  That's more or less all I'm after here.  It's a minefield already, and if it wasn't for testing requirements I wouldn't want it at all.

> add a wrapper mock around the call to get previous backups

That sounds like a healthy alternative overall.  Also, like you said, heavy.  I'll make a tech debt ticket for this, but am going to carry on with the current implementation.